src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/TestHelpers/TestIVsAsyncFileChangeEx.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.VisualStudio.Shell
+{
+    public class TestIVsAsyncFileChangeEx : IVsAsyncFileChangeEx
```

Comment:
This is a mock, can we name it as such and move it?

Reply:
Yep. I wasn't sure about this when I made it, but I guessed that mocks were exclusively for Moq mocks :)